roles/pulp/vars/Debian.yml

```diff
@@ -2,6 +2,7 @@
 pulp_preq_packages:
   - python3
   - python3-venv
+  - postgresql-11-debversion
```

Comment:
This package needs packaging on the RPM side.

Reply:
@mdellweg I am a Fedora packager. I can help you with this.

Reply:
libapt-pkg5.0 would be a new dependency.

We can use the mageia 4.0 RPMs as a starting point:
https://mageia.pkgs.org/cauldron/mageia-core-release-i586/libapt-pkg4-0.5.15lorg3.94-33.mga8.i586.rpm.html

Reply:
I believe i have seen this package. Is mageia not a direct descendent of mandrake? I think, they ported apt to use rpm, and consequently patched all the debian package specific stuff away.
Therefore i was persuing another way. I looked for the single function we need, and patched that one into the postgres-extension:
https://github.com/ATIX-AG/foreman-packaging/blob/add_postgresql_debversion/packages/katello/postgresql-debversion/0001-Copy-relevant-code-from-apt-pkg-to-ease-packaging.patch
This drops the dependency on libapt completely.

Reply:
Started an effort to write a spec file for this:
https://github.com/theforeman/foreman-packaging/pull/4540

Still i am not really sure that is the place to make it available for all pulp users.